### PR TITLE
Add changelog.md

### PR DESCRIPTION
**Pull request by collindutter**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```
Comment (cjkindel) [inline CHANGELOG.md:4]: Do we want to use 1.0.0 or [1.1.0](https://keepachangelog.com/en/1.1.0/)?

Comment (collindutter) [reply CHANGELOG.md:4]: Lol I copied the text from their 1.1.0 site which references 1.0.0.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+### Fixed
+- `MongoDbAtlasVectorStore` namespace not being used properly when querying. 
+- Miscellaneous type errors throughout the codebase.
+- Remove unused section from `ToolTask` system prompt template.
+- Structure execution args being cleared after run, preventing inspection of the Structure's `input_task`'s `input`.
+- Unhandled `SqlClient` exception. Thanks @michal-repo!
+
+### Changed
+- **BREAKING**: Rename `input_template` field to `input` in Tasks that take a text input.
+- **BREAKING**: Rename `BedrockTitanEmbeddingDriver` to `AmazonBedrockTitanEmbeddingDriver`.
```
Comment (shhlife) [inline CHANGELOG.md:32]: Didn't we also have breaking changes for `AmazonBedrockStableDiffusionImageGenerationModelDriver` and `AmazonBedrockTitanImageGenerationModelDriver`?

they're now just `BedrockStableDiffusionImageGenerationModelDriver` &  `BedrockTitanImageGenerationModelDriver`

File: CHANGELOG.md (hunk 1 above)
Comment (cjkindel) [reply CHANGELOG.md]: lol, time for 1.1.1

Comment (collindutter) [reply CHANGELOG.md]: https://keepachangelog.com/en/1.1.1/ is a dead link 💯

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+## [0.22.0] - 2024-01-11
+
+### Added
+- `PromptImageGenerationEngine` for generating images from text prompts. 
+- `VariationImageGenerationEngine` for generating variations of an input image according to a text prompt.
+- `InpaintingImageGenerationEngine` for modifying an input image according to a text prompt within the bounds of a mask defined by a mask image. 
+- `OutpaintingImageGenerationEngine` for modifying an input image according to a text prompt outside the bounds of a mask defined by a mask image.
+- `ImageArtifact` for storing image data; used heavily by the image Engines, Tasks, and Drivers.
+- `ImageLoader` for loading images files into `ImageArtifact`s.
+- Support for all Tokenizers in `OpenAiChatPromptDriver`, enabling OpenAi drop-in clients such as TogetherAi.
```
Comment (SavagePencil) [inline CHANGELOG.md:16]: "enabling OpenAi drop-in" -> "enabling OpenAI drop-in"
"TogetherAI" -> "Together AI"

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- `AmazonSageMakerEmbeddingDriver` for using Amazon SageMaker to generate embeddings. Thanks @KaushikIyer16!
+- Claude 2.1 support in `AnthropicPromptDriver` and `AmazonBedrockPromptDriver` via `BedrockClaudePromptModelDriver`.
+- `CodeExecutionTask` for running Python functions without an LLM.
```
Comment (SavagePencil) [inline CHANGELOG.md:19]: if it's for running Python, then shouldn't it have been named PythonTask :) . I'd say "for executing code as a task without the need for an LLM."

File: CHANGELOG.md (hunk 2 above)
Comment (shhlife) [reply CHANGELOG.md:32]: oh yeah, this means that ImageGenerationTask is now PromptImageGenerationTask (breaking change) and ImageGenerationEngine is now PromptImageGenerationEngine (breaking change).

**Review by SavagePencil — APPROVED** [review]
approved, pending addressing other people's comments :)

**Pull request merged**